### ISSUE TITLE
Can I use the 3.0 branch in production code?

### ISSUE THREAD

**Issue by mkoryak** [mapstraction/mxn#301]
The last commit in master was one year ago. I noticed that there is a lot of activity in the '3.0' branch. Can I use that branch in prod code?

Would I be better off using the code in master and waiting for the 3.0 release?

**Comment by vicchi** [comment]
You're absolutely right that there's a lot of activity going on on the `release-3.0` branch (and also on several other development branches that use `release-3.0` as their ref).

It's great that you're keeping an eye on what's going on with Mapstraction but for now, I'd really recommend that you continue to use the old v2.x release on `master`.

The code base for Mapstraction is going through an awful lot of changes, a lot of them that will end up breaking if used with code that assumes Mapstraction v2.x. Also some of these changes are documented in the code, most are not and none of them are documented on mapstraction.com or on the project's wiki. Also, also, these changes themselves are changing. In other words, the v3.0 branches are far from being stable.

Both myself and @gilesc50, together with everyone else who's been submitting pull requests, are doing all we can with the time we have to get v3.0 finished, stable and documented. By all means keep an eye on the v3.0 branches but I think using them in production is unwise right now.

**Comment by mkoryak** [comment]
Are you aware that the leaflet example seems to be broken on master?
adding marker results in:

`Error: iconUrl not set in Icon options (see the docs).
...=t,this._map&&this._updateOpacity(),this},_updateOpacity:function(){o.DomUtil.se...
leaflet.js (line 7)`

this is in mxn/examples/leaflet.html

**Comment by vicchi** [comment]
Yes, absolutely. See Issue #285 for more background context.

**Comment by mkoryak** [comment]
Can I make a suggestion?

Add something to the master README.MD about the 2.1 branch and about the progress being made on 3.0.

For a new user just starting to use this lib, setting up this lib was a bit confusing. I had to figure out that there was activity on this project, just not on the master branch. I didn't know right away to pull down the 2.1 release branch, and learned the hard way.

I did go through all the issues before starting to use the lib, but missed the 2.1 branch issue. 

Thanks for your work on this!
